README.md

```diff
@@ -125,6 +125,26 @@ import { renderToStaticMarkup } from 'react-dom/server';
 import parse from 'html-react-parser';
 import domToReact from 'html-react-parser/lib/dom-to-react';
```

Comment:
Since we're updating this code block, do you mind remove this line and updating the line above?
```js
import parse, { domToReact } from 'html-react-parser';
```

Reply:
sure, updated